### DISCUSSION TITLE
Display section title in cross reference?

### DISCUSSION THREAD

**Discussion by th0ger** [quarto-dev/quarto-cli#9085] [Q&A] [answered]
### Description

With [cross references for section](https://quarto.org/docs/authoring/cross-references.html#sections), is it possible to show the section title instead of section number in the link?

**Comment by mcanouil** [comment, marked as answer]
Closing as duplicate.

- https://github.com/quarto-dev/quarto-cli/issues/4532

PS: please search for issues/discussion before opening new ones. Here the issue title is nearly word for word your request.
Anyway, thanks for the feedback.

**Reply by th0ger** [reply]
Sure, I did search both issues and discussions, but apparently missed the open issue. Maybe I've used different search phrases.

**Reply by mcanouil** [reply]
FYI, I knew the issue existed but I searched using your title, so nothing really fancy or specific.

![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/8f24076e-4295-41d4-8279-f1dd56d0b2e3)
